string object_name
---
bool sent_goal